[FEATURE] Expose Playwright async/await flavor

I need to run `aiohttp` server and use `playwright` in tests at the same time.
Am I right that `pytest-playwright` right now can't help me with that?

It provides fixtures only for sync API. But I need async API.
So I have to do something like that.
```py
from aiohttp import web
from playwright.async_api import async_playwright

async def index_html(request):
    return web.Response(body="<html><body>Hello world</body></html>", content_type="text/html")

async def test_without_page_fixture_passes(aiohttp_server):
    app = web.Application()
    app.router.add_get("/", index_html)
    server = await aiohttp_server(app)
    async with async_playwright() as p:
        browser = await p.chromium.launch()
        context = await browser.new_context()
        page = await context.new_page()
        await page.goto(str(server.make_url("/")))
        assert await page.text_content("body") == "Hello world"
```

> Exactly, currently the pytest plugin does only provide the sync APIs. So you need to launch browser etc. yourself, something similar to https://github.com/microsoft/playwright-python/blob/master/tests/async/conftest.py
> 
> Shall I rename this issue title into a feature request for providing also the async flavor?

@mxschmitt I'm on Playwright v1.17 and I did try to copy-paste all the fixtures in the file you pointed here (except the `utils`), but then I got:
```
E       fixture 'launch_arguments' not found
>       available fixtures: _verify_url, base_url, browser, browser_channel, browser_context_args, browser_factory, browser_name, browser_type, browser_type_launch_args, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, context, context_factory, delete_output_dir, device, doctest_namespace, event_loop, is_chromium, is_firefox, is_webkit, launch_browser, monkeypatch, page, password, playwright, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, report_name, selectors, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, username
>       use 'pytest --fixtures [testpath]' for help on them.
```

I quickly fixed it by changing `launch_arguments` to `browser_type_launch_args`, but then again, I got:
```
playwright = <async_generator object playwright at 0x111459d40>, browser_name = 'chromium'

    @pytest.fixture(scope="session")
    def browser_type(playwright, browser_name: str):
        if browser_name == "chromium":
>           return playwright.chromium
E           AttributeError: 'async_generator' object has no attribute 'chromium'

conftest.py:20: AttributeError
```

I'm pretty new to Python and pytest but I guess `async_generator` of `playwright_object` instead of `playwright_object` itself being "injected" to `browser_type` – any pointer on how I should remedy this? And is this `async` gonna be planned to be supported in the end? 😅  Thank you.

--- a/docs/src/test-runners-python.md
+++ b/docs/src/test-runners-python.md
@@ -259,3 +259,18 @@ def test_bing_is_working(page):
 ## Deploy to CI
 
 See the [guides for CI providers](./ci.md) to deploy your tests to CI/CD.
+
+## Async Fixtures
+
+If you want to use async fixtures, you can use the [`pytest-playwright-asyncio`](https://pypi.org/project/pytest-playwright-asyncio/) plugin.
+Make sure to use `pytest-asyncio>=0.24.0` and make your tests use of [`loop_scope=sesion`](https://pytest-asyncio.readthedocs.io/en/latest/how-to-guides/run_session_tests_in_same_loop.html).
+
+```python
+import pytest
+from playwright.async_api import Page
+
+@pytest.mark.asyncio(loop_scope="session")
+async def test_foo(page: Page):
+    await page.goto("https://github.com")
+    # ...
+```

